monsoft-crm-app/bases/ai/utils/ai-model-to-sdk-model-provider.util.ts

```
@@ -0,0 +1,43 @@
+import { createAnthropic } from '@ai-sdk/anthropic';
+import { createOpenAI } from '@ai-sdk/openai';
+
+import { getCoreConf } from '@conf/core/providers/server';
+
+import { AiModel } from '../enums';
+
+import { aiModelToProvider } from './ai-model-to-provider.util';
+
+import { Error, Success } from '@errors/utils';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)